uplc.md

```diff
@@ -35,7 +34,14 @@ module UPLC-SYNTAX
                          | "greaterThanInteger"
                          | "greaterThanEqualsInteger"
                          | "equalsInteger"
-                         | "sha3_256"
+                         | "appendByteString"
+			 | "consByteString"
```

Comment:
looks like there are tabs on each indentation start from this line to the end of the declaration